packages/repository-tests/src/crud/relations/fixtures/repositories/product.repository.ts

```diff
@@ -3,9 +3,8 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {DefaultCrudRepository, juggler} from '../../..';
+import {DefaultCrudRepository, juggler} from '@loopback/repository';
 import {Product, ProductRelations} from '../models/product.model';
-export {Product};
 
 export class ProductRepository extends DefaultCrudRepository<
```

Comment:
As I commented earlier, all model-specific repository classes must inherit from `repositoryClass` provided to the test suite by the caller. In the (near) future, we will want to run the shared test suite against `CrudRepositoryImpl` in addition to `DefaultCrudRepository`. See https://github.com/strongloop/loopback-next/blob/6ad0bb59c71ea2356cb951da786bdbff246b47e7/packages/repository/src/repositories/repository.ts#L248